Zeplin.sketchplugin/Contents/Sketch/manifest.json

```diff
@@ -48,16 +76,28 @@
         "name": "Shortcut Help",
         "identifier": "shortcut-help",
         "shortcut": "cmd e",
-        "script": "export.cocoascript",
+        "script": "utils.cocoascript",
```

Comment:
Here's an idea: We can move the methods in `utils` to `export` and rename `export` to something like`main` or `index`. In that case, we can rename `helpers` to `utils`, which would make more sense. 😅